docs/content/ingestion/batch-ingestion.md

```diff
@@ -198,7 +198,7 @@ The tuningConfig is optional and default parameters will be used if no tuningCon
 |-----|----|-----------|--------|
 |workingPath|String|the working path to use for intermediate results (results between Hadoop jobs).|no (default == '/tmp/druid-indexing')|
 |version|String|The version of created segments.|no (default == datetime that indexing starts at)|
-|leaveIntermediate|leave behind files in the workingPath when job completes or fails (debugging tool).|no (default == false)|
+|leaveIntermediate|Boolean|leave behind files in the workingPath when job completes or fails (debugging tool).|no (default == false)|
```

Comment:
I was going to suggest putting this one right after cleanupOnFailure, just was too slow about it.